### ISSUE TITLE
wrong org...

### ISSUE THREAD

**Issue by PythonCHB** [UWPCE-PythonCert-ClassRepos/PythonCert220Assign#1]
This should be in the ClassRepos org:

https://github.com/UWPCE-PythonCert-ClassRepos

unless it's intended to be a template for instructors -- though even then, it should probably go there. And there is a template there already if you want:

https://github.com/UWPCE-PythonCert-ClassRepos/ClassRepoTemplate

Also -- I'm not sure you want students to clone and rename -- rather, the instructor can create one for the particular class, and then the students can fork that.

Unless you have a new workflow in mind for course 2 -- in which case, it is written down somewhere?

**Comment by pythonsavvy** [comment]
org changed. STudent procedures are in the class which we will push soon

**Comment by pythonsavvy** [comment]
@PythonCHB thanks for the help, interested in your comments on this.  I just found this https://uwpce-pythoncert.github.io/PythonCertDevel/modules/Git.html#setting-up-your-fork-of-the-class-repository and had not been aware of it before. The TA did all the grading in the last class. What I would like to do for py220 is to give each student their own, personal  repo that I can setup and they can simply push and pull to / from. I had assumed that was the way we worked, as it is representative of how we work on projects at EXPD with GitLab;  it keeps it simple, and still allows sharing. Also, it makes it easier for the student and instructor to both run the automated tests to grade assignments that I created given my assumption - there is one lessons package that gets pytested, linted etc, but can go down to just one module. However, I now realize it is different and causes a proliferation of repos. Perhaps I should create an org for the class, and create the student repos per student under there? Dont want to be different for different sake, but I think this makes setup and maintenance very simple (I use the API to create the repos for example).
